src/core/components/subscription_manager.js

```diff
@@ -419,15 +419,30 @@ export default class {
 
   _startSubscribeLoop() {
     this._stopSubscribeLoop();
+    let presenceState = {};
```

Comment:
Technically these can be `const` but it's fine to keep the same pattern for now.